src/main/Server.java

```diff
@@ -0,0 +1,31 @@
+import clientHandler.ClientHandlerThread;
+import clientInteractors.ClientInteractorInterface;
+import clientInteractors.TCPClientInteractor;
+
+import java.io.IOException;
+
+public class Server {
+
+    public static void main(String[] args) {
+	    runServer();
+    }
+
+    private static void runServer() {
+        while(true){
+            ClientInteractorInterface clientInteractor = setUpInteraction();
+            new Thread(new ClientHandlerThread(clientInteractor)).start();
```

Comment:
Then your server we be somewhat 'frozen' to direct input: The main thread is completely focused on accepting connections. I suppose this is not a problem if your server will never do anything else (is this the case?).